ui/app/src/modules/Preview/Tools/AudiencesPanel.tsx

```diff
@@ -225,158 +240,29 @@ export default function AudiencesPanel() {
-                  <MenuItem value={possibleValue.value} key={index}>{possibleValue.value}</MenuItem>
-                ))
-              ) : (null)
-            }
-          </Select>
-          <FormHelperText>{property.helpText}</FormHelperText>
-        </AudiencesControl>
-      );
-    case "checkbox-group":
-      const valuesArray = nnou(modelValue) ? modelValue.split(',') : [];
-
-      return (
-        <AudiencesControl property={property}>
-          <>
-            {
-              property.values ? (
-                property.values.map((possibleValue: any, index: number) => (
-                  <FormControlLabel
-                    key={index}
-                    htmlFor={property.id}
-                    control={
-                      <Checkbox
-                        color="primary"
-                        checked={valuesArray.includes(possibleValue.value)}
-                        onChange={handleInputChange(property.id, possibleValue.value, valuesArray)}
-                        disabled={modelApplying}
-                      />
-                    }
-                    label={possibleValue.value}/>
-                ))
-              ) : (null)
-            }
-            <FormHelperText>{property.helpText}</FormHelperText>
-          </>
-        </AudiencesControl>
-      );
-    case "date-time":
-      return (
-        <AudiencesControl property={property}>
-          <>
-            <DateTimePicker
-              initialDate={modelValue}
-              timezone={modelTimezone}
-              onChange={dateTimePickerChange(property.id)}
-              disabled={modelApplying}/>
-            <FormHelperText>{property.helpText}</FormHelperText>
-          </>
-        </AudiencesControl>
-      );
-    default:
-      return (
-        <AudiencesControl property={property}>
-          <TextField
-            id={property.id}
-            type="text"
-            name="input"
-            placeholder="auto"
-            fullWidth
-            value={modelValue}
-            helperText={property.helpText}
-            onChange={handleInputChange(property.id)}
-            disabled={modelApplying}
-          />
-        </AudiencesControl>
-      );
-  }
-}
-
-interface AudiencesControlProps {
-  property: ContentTypeField;
+interface AudiencesFormSectionProps {
+  field: ContentTypeField;
   children: any;
 }
 
-function AudiencesControl(props: AudiencesControlProps) {
+function AudiencesFormSection(props: AudiencesFormSectionProps) {
   const classes = useStyles({});
 
-  const { property, children } = props;
+  const { field, children } = props;
 
   return (
     <Grid item xs={12}>
       <FormControl className={classes.formControl}>
         <InputLabel
           className={classes.InputLabel}
           focused={true}
-          htmlFor={property.id}
+          htmlFor={field.id}
         >
-          {property.name}
+          {field.name}
         </InputLabel>
         {children}
       </FormControl>
```

Comment:
The FormControl and InputLabel need to be on the concrete Control as some controls won't be a simple form field (in the context of the form engine).

The FormSection's job is just add base structure - specific to the panel UI - to wrap the control.